### PR TITLE
EKS workshop PDP-1322

### PR DESCRIPTION
**Pull request by nashtapu-tibco**

### CODE REVIEW

**Review by syan-tibco — APPROVED** [review]

File: dp-cluster-workshop/eks/README.md [hunk 1]
```diff
@@ -205,6 +208,26 @@ storageClass:
...
+      opentelemetry-operation-name: HTTP $request_method $service_name $uri
+      opentelemetry-trust-incoming-span: "true"
+      otel-max-export-batch-size: "512"
+      otel-max-queuesize: "2048"
+      otel-sampler: AlwaysOn
+      otel-sampler-parent-based: "false"
+      otel-sampler-ratio: "1.0"
+      otel-schedule-delay-millis: "5000"
+      otel-service-name: nginx-proxy
+      otlp-collector-host: otel-userapp.${DP_NAMESPACE}.svc
```
Comment (akusingh-tibco) [inline dp-cluster-workshop/eks/README.md:227]: @nashtapu-tibco we should export the value of DP_NAMESPACE right ?

Comment (nashtapu-tibco) [reply dp-cluster-workshop/eks/README.md]: good point, please review now.

File: dp-cluster-workshop/eks/README.md [hunk 2]
```diff
@@ -208,26 +211,28 @@ storageClass:
     enabled: ${DP_EFS_ENABLED}
     parameters:
       fileSystemId: "${DP_EFS_ID}"
-ingress-nginx:
-  controller:
-    config:
-      use-forwarded-headers: "true"
```
Comment (akusingh-tibco) [inline dp-cluster-workshop/eks/README.md]: @nashtapu-tibco we should keep the line no 214 (needed by BWCE apps) and comment the below ones

Comment (nashtapu-tibco) [reply dp-cluster-workshop/eks/README.md]: I was checking, it worked without that config as well.

Comment (nashtapu-tibco) [reply dp-cluster-workshop/eks/README.md]: Changes are done.

**Pull request merged**